metadetect/vis.py

```diff
@@ -60,29 +60,41 @@ def show_mbexp(
     The axis used for plotting
     """
     from astropy.visualization.lupton_rgb import AsinhMapping
+    from astropy.visualization import (
+        AsinhStretch,
+        imshow_norm,
+    )
     import scarlet
```

Comment:
This is a very heavy dep for showing a color image.

Reply:
I wanted the visualizations to match Scarlet so the devs could quickly grok my images.

This is lsst only code; I assumed (incorrectly?) that this was a dep already for the stack.  Happy to rename this to lsst_vis.py